lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -2621,7 +2621,14 @@ ParseResult InvokeOp::parse(OpAsmParser &parser, OperationState &result) {
 }
 
 void InvokeOp::print(OpAsmPrinter &p) {
```

Comment:
Can you add a test for this please, maybe in round-trip?

Reply:
Round-trip? From SCF to Calyx and from Calyx back to?

Reply:
There should be a round-trip.mlir test file (or something similar, not on my laptop) that just verified that printing and parsing works as expected

Reply:
done!